### DISCUSSION TITLE
10  AI Acceleration – Machine Learning Systems

### DISCUSSION THREAD

**Discussion by giscus** [harvard-edge/cs249r_book#348] [General]
# 10  AI Acceleration – Machine Learning Systems

https://mlsysbook.ai/contents/hw_acceleration/hw_acceleration.html

**Comment by emmanuel2406** [comment]
I enjoyed the emerging technologies section. It's often easy to take the classical computing architecture for granted, and quite inspiring to read about these new approaches (neuromorphic, quantum, analog, optical). Even though it might still be a while before there is a paradigm shift of this magnitude, it doesn't stop us from dreaming about it.
